### DISCUSSION TITLE
FSK4_Transmit example and debug code -13

### DISCUSSION THREAD

**Discussion by radiohound** [jgromes/RadioLib#1356] [General]
Hello,

I am trying to run the FSK4_Transmit example, and I get the following debug code. My device is the STM32WLE5. This should have a SX1262, but debug reports a SX1261. Does anyone know what code -13 means?

Thank you,

Walter

```
[SX1278] Initializing ... RLB_DBG: 
RadioLib Info
Version:  "7.1.0.0"
Platform: "Arduino STM32 (official)"
Compiled: "Dec 20 2024" "06:45:16"
RLB_DBG: Found SX126x: RADIOLIB_SX126X_REG_VERSION_STRING:
RLB_DBG: 00000320: 53 58 31 32 36 31 20 54 4b 46 20 31 41 31 31 00  SX1261 TKF 1A11.
RLB_DBG: 
RLB_DBG: M      SX126x
failed, code -13
```

**Comment by jgromes** [comment]
You can find list of all status codes [here](https://jgromes.github.io/RadioLib/group__status__codes.html), linked from multiple places within this repository. We also have an [online status code decoder](https://radiolib-org.github.io/status_decoder/decode.html).
